app/screens/home/index.tsx

```diff
@@ -136,7 +146,10 @@ export function HomeScreen(props: HomeProps) {
     }, []);
 
     return (
-        <>
+        <View
+            style={styles.flex}
+            nativeID={SecurityManager.getShieldScreenId(Screens.HOME, true)}
```

Comment:
The first time we render this we may not have the SecurityManager initialized (the effect has not run yet), which means we may not have an `activeServer`, which has implications for the shieldScreenId.

Is this a problem?

Reply:
is fine as for this one, we are forcing the shield ID, that is what the `true` as a second argument is for.

the function does this `if ((this.activeServer && this.isScreenCapturePrevented(this.activeServer)) || force) {` and force in this case is true, so the ID is applied.